### ISSUE TITLE
DOCKER_OPTS not loaded on 15.10

### ISSUE THREAD

**Issue by dggreenbaum** [angstwad/docker.ubuntu#80]
It seems that the DOCKER_OPTS variable either isn't being loaded or isn't being read when systemd goes to start Docker. This results in, for example, being unable to expose the docker daemon on a port.

The referenced PR is a hack to get around this problem, though I suspect the root cause is PEBKAC. If there is a better way around this I'd love to throw my PR away.

I am using the ubuntu/wily64 Vagrant box. For reference, my playbook.yml is as follows:

```
- hosts: all
  sudo: true
  roles:
    - role: roles/docker.ubuntu
      docker_opts: >
        -H unix://
        -H tcp://0.0.0.0:2375
        --log-level=debug
```

**Comment by jonseymour** [comment]
@dggreenbaum Does the issue resolve itself if you manually restart docker?

**Comment by angstwad** [comment]
Probably resolved by #90; reopen if not resolved.

**Comment by jonseymour** [comment]
@angstwad I think my fix #90 only applies to Ubuntu releases earlier than 15.04 since later releases will use a different set of tasks.
